ApplicationServerDeployment/lambda/alexa/alexa_lambda_handler.py

```diff
@@ -0,0 +1,54 @@
+from discovery import DiscoveryControl
+from power_control import PowerControl
+from report_state import ReportState
+
+from utils import *
+import logging
+import json
+from response import AlexaResponse
+
+logger = logging.getLogger()
+if logger.handlers:
+	for handler in logger.handlers:
+		logger.removeHandler(handler)
+logging.basicConfig(format='%(asctime)s %(message)s', level=logging.ERROR)
+
+def lambda_handler(event, context):
+
+	if 'directive' not in event:
+		logger.error("Missing key: directive, Is the request a valid Alexa Directive?")
+		return False
+
+	#logger.info(json.dumps(event))

```

Comment:
Can we remove the log line if not needed or change it to debug ?

Reply:
@sagarraj92 I have uncommented this on the most recent commit [1ccfdff](https://github.com/aws-samples/aws-iot-core-for-amazon-sidewalk-sample-app/pull/48/commits/1ccfdff52f11a223051da16cb0ef0cf86365cf46) , and also enabled the INFO logger as I believe this app is a sample stage and not meant for production it will help developers/device makers get visibility into incoming requests from Alexa to better understand the req/resp structure. With this expectation do you think we can let it be?